klaviyo/klaviyo-swift-sdk
# [CHNL-13246] add webview popup chrome

# Description

This adds the "chrome" around the WebView that will be used for the IAM content. The new `KlaviyoWebWrapperViewController` wraps the `KlaviyoWebViewController` and adds an inset, drop shadow, background blur, and/or background tint. These styles can all be customized using the `KlaviyoWebWrapperStyle` struct. 

This PR addresses [CHNL-13246](https://klaviyo.atlassian.net/browse/CHNL-13246).

# Check List


# Manual Test Plan

Run the Xcode Preview within the `KlaviyoWebWrapperViewController` file and validate that the view looks and works as expected.

# Supporting Materials

Here are some examples of what the `KlaviyoWebWrapperViewController` looks like with different styles applied:

<table>
    <tr><th>Blurred background</th><th>Tinted background</th></tr>
    <tr><td><img width="306" alt="Screenshot 2024-10-28 at 13 15 33" src="https://github.com/user-attachments/assets/b0d4872b-a816-4854-b1ac-23c5648e6761"></td><td><img width="307" alt="Screenshot 2024-10-28 at 13 15 48" src="https://github.com/user-attachments/assets/c1234b8c-fdf0-42c4-8970-5b76c07acc5f"></td></tr>
</table>

[CHNL-13246]: https://klaviyo.atlassian.net/browse/CHNL-13246?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
